### PR TITLE
Trials branch

### PR DESCRIPTION
**Pull request by ilyassatlassi**
Create a Survey form Using Html&Css

### CODE REVIEW

**Review by Favourezeugwa — COMMENTED** [review]
Hi @ilyassatlassi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ ❌ 

- You have submitted a project, that hasn't been reviewed. Please kindly review this Pull request, after reviewing it yourself, you can go ahead and submit the request review on your dashboard. 

- A code reviewer will review how you have reviewed your own code. ✔️ 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+</head>
+
+<body>
+  <div class="bg">
+    <h1 id="title">
+      freeCodeCamp Survey Form
+    </h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
```
Comment (ilyassatlassi) [inline index.html:20]: You can use a semantic tag like nav ,Describing the contents of the page more semantically can have many benefits, such as:

Making code easier for people to navigate when updating features or fixing bugs.
Giving your page a higher Search Engine Optimization (SEO) score, which makes your content more visible on search engines.
Allowing users with special needs to access your website using alternative means such as screen readers, which are devices that read the contents of a webpage aloud to users who have visual impairments.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Definitely
+        </label>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Maybe
+        </label>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Not sure
```
Comment (ilyassatlassi) [inline index.html:64]: You can use a semantic tag like: section, Describing the contents of the page more semantically can have many benefits, such as:

Making code easier for people to navigate when updating features or fixing bugs.
Giving your page a higher Search Engine Optimization (SEO) score, which makes your content more visible on search engines.
Allowing users with special needs to access your website using alternative means such as screen readers, which are devices that read the contents of a webpage aloud to users who have visual impairments.

File: index.html (hunk 1 above)
Comment (Felix45) [reply index.html:20]: [ Required ] The purpose of giving a code review is to improve the project so it's important that you explain to the student the semantic tag that he should use and the benefit that semantic tag will have on his code base.

Please consider editing this review while factoring in my suggestions.

File: index.html (hunk 2 above)
Comment (Felix45) [reply index.html:64]: [ Required ] The purpose of giving a code review is to help improve the project so it's important that you explain to the student the semantic tag that he should use and the benefit that semantic tag will have on his code base.

Please consider editing this review while factoring in my suggestions.

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @ilyassatlassi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] All linter checks are passing
- [ ] Used GitHub review changes inline comments feature 

almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ optional ] Please consider providing a summary of the whole review, this makes your review  more professional and also helps the student know what he did well and the areas he needs to work on to improve his project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ilyassatlassi) [reply index.html:20]: Describing the contents of the page more semantically can have many benefits, such as:

Making code easier for people to navigate when updating features or fixing bugs.
Giving your page a higher Search Engine Optimization (SEO) score, which makes your content more visible on search engines.
Allowing users with special needs to access your website using alternative means such as screen readers, which are devices that read the contents of a webpage aloud to users who have visual impairments.

File: index.html (hunk 2 above)
Comment (ilyassatlassi) [reply index.html:64]: Describing the contents of the page more semantically can have many benefits, such as:

Making code easier for people to navigate when updating features or fixing bugs.
Giving your page a higher Search Engine Optimization (SEO) score, which makes your content more visible on search engines.
Allowing users with special needs to access your website using alternative means such as screen readers, which are devices that read the contents of a webpage aloud to users who have visual impairments.

**Review by aimalamiri — APPROVED** [review]
Hi @ilyassatlassi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

Good job for using the GitHub review comments 💯
Comments are respectful and professional 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**